script.js

```diff
@@ -0,0 +1,338 @@
+  { code: '53', text: '%', location: '0' }, { code: '54', text: ':', location: '0' }, { code: '55', text: '?', location: '0' },
+  { code: '56', text: '*', location: '0' }, { code: '57', text: '(', location: '0' }, { code: '48', text: ')', location: '0' },
+  { code: '189', text: '_', location: '0' }, { code: '187', text: '+', location: '0' }, { code: '8', text: 'Backspace', location: '0' },
+  { code: '9', text: 'Tab', location: '0' }, { code: '81', text: 'Й', location: '0' }, { code: '87', text: 'Ц', location: '0' },
+  { code: '69', text: 'У', location: '0' }, { code: '82', text: 'К', location: '0' }, { code: '84', text: 'Т', location: '0' },
+  { code: '89', text: 'Н', location: '0' }, { code: '85', text: 'Г', location: '0' }, { code: '73', text: 'Ш', location: '0' },
+  { code: '79', text: 'Щ', location: '0' }, { code: '80', text: 'З', location: '0' }, { code: '219', text: 'Х', location: '0' },
+  { code: '221', text: 'Ъ', location: '0' }, { code: '46', text: 'Delete', location: '0' }, { code: '20', text: 'CapsLock', location: '0' },
+  { code: '65', text: 'Ф', location: '0' }, { code: '83', text: 'Ы', location: '0' }, { code: '68', text: 'В', location: '0' },
+  { code: '70', text: 'А', location: '0' }, { code: '71', text: 'П', location: '0' }, { code: '72', text: 'Р', location: '0' },
+  { code: '74', text: 'О', location: '0' }, { code: '75', text: 'Л', location: '0' }, { code: '76', text: 'Д', location: '0' },
+  { code: '186', text: 'Ж', location: '0' }, { code: '222', text: 'Э', location: '0' }, { code: '13', text: 'Enter', location: '0' },
+  { code: '16', text: 'Shift', location: '1' }, { code: '220', text: '\\', location: '0' }, { code: '90', text: 'Я', location: '0' },
+  { code: '88', text: 'Ч', location: '0' }, { code: '67', text: 'С', location: '0' }, { code: '86', text: 'М', location: '0' },
+  { code: '66', text: 'И', location: '0' }, { code: '78', text: 'Т', location: '0' }, { code: '77', text: 'Ь', location: '0' },
+  { code: '188', text: 'Ю', location: '0' }, { code: '190', text: 'Ю', location: '0' }, { code: '191', text: '/', location: '0' },
+  { code: '38', text: '▲', location: '0' }, { code: '16', text: 'Shift', location: '2' }, { code: '17', text: 'Ctrl', location: '1' },
+  { code: '91', text: 'Win', location: '0' }, { code: '18', text: 'Alt', location: '1' }, { code: '32', text: ' ', location: '0' },
+  { code: '18', text: 'Alt', location: '2' }, { code: '17', text: 'Ctrl', location: '2' }, { code: '37', text: '◄', location: '0' },
+  { code: '40', text: '▼', location: '0' }, { code: '39', text: '►', location: '0' }];
+
+const keyboard = document.createElement('div');
+keyboard.classList.add('keyboard');
+
+const wrapper = document.createElement('div');
+wrapper.classList.add('wrapper');
+
+const description = document.createElement('div');
+description.classList.add('description');
+description.innerHTML = '<div>Ctrl + Alt  - change language</div><div><i>Windows OS</i></div>';
+
+document.body.append(wrapper);
```

Comment:
It would be better to manipulate with your elements inside JS and then after all elements are created append your `wrapper` once inside DOM.
Every manipulation with DOM using Element.append trigger page life-cycle phases which negatively affects performance.
I mean to append your wrapper inside document.body at the end of the scripts file when all buttons and needed elements created inside JS.